specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -260,6 +260,10 @@
                 "type": "integer",
                 "format": "int32",
                 "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              },
+              "Azure-AsyncOperation": {
```

Comment:
We should not manually change the swagger json. All swagger json changes should come from typespec.

Reply:
@SahilArora92 - Please update the Tsp files for correct response for 200. It should be one of the terminal states - Succeeded or Failed.

Also Add -
"Azure-AsyncOperation": {
                "description": "URL for checking the ongoing status of the operation.\nTo get the status of the asynchronous operation, send a GET request to the URL in Azure-AsyncOperation header value.\n\nFor more info: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations",
                "type": "string"
              } - Simlar to this in the swagger or TSP file, as it's missing.

Reply:
},
    "201": {
      "headers": {
        "Azure-AsyncOperation": "https://contoso.com/operationstatus",
        "Retry-After": 30
      },
      "body": {
        "properties": {
          "provisioningState": "Succeeded", - This should be Creating for Put and Updating for Patch.

Reply:
@SahilArora92 
Also, the 200, 201 response body should be matching the request submitted (for Put, Patch). Let's make sure that it has correct values.

Fleets_ListByResourceGroup -  "provisioningState": "Creating", Which may correct. Please check it.
"Fleets_ListBySubscription" - The response look too short as compared to Fleets_ListByResourceGroup. Please check it.